ui/ui-frontend/projects/vitamui-library/src/app/modules/components/file-selector/file-selector.component.html

```diff
@@ -15,8 +27,8 @@
 </div>
 
 <ng-template #defaultFileList>
-  <div class="vitamui-chip-list">
-    <div *ngFor="let file of files" class="vitamui-chip">
+  <div class="vitamui-chip-list defaultFileList">
```

Comment:
Il n'y a pas de classe `defaultFileList`
```suggestion
  <div class="vitamui-chip-list">
```